src/services/Node/index.js

```diff
@@ -18,6 +18,13 @@ class Node {
   _balance = new BehaviorSubject(0);
   balance$ = this._balance.asObservable();
 
+  /**
+   * @type {BehaviorSubject}
+   * @private
+   */
+  _fee = new BehaviorSubject(0);
+  fee$ = this._fee.asObservable();
```

Comment:
great use of the behavior subject and its associated methods for displaying the fee for any given tool. The logic here is sound: the user will only ever be submitting one transaction at a time, and therefore, we only need to keep track of one fee state at a time for any given kind of transaction.